x-pack/legacy/plugins/siem/public/components/load_more_table/index.tsx

```diff
@@ -188,7 +188,7 @@ export class LoadMoreTable<T, U, V, W, X, Y, Z, AA, AB> extends React.PureCompon
     return (
       <Panel
         data-test-subj={dataTestSubj}
-        loading={{ loading }}
+        loading={loading ? 1 : 0}
```

Comment:
Had a lot of fun here with styled component, any suggestions are welcome.
used to rename the props to `isLoading`, then I got:

<img width="1680" alt="Screenshot 2019-09-20 at 00 46 24" src="https://user-images.githubusercontent.com/6295984/65288899-7216d680-db40-11e9-8e10-37641763feac.png">

Besides, somehow it converts boolean to string, that's why I'm changing the type of loading here

<img width="1680" alt="Screenshot 2019-09-20 at 00 46 37" src="https://user-images.githubusercontent.com/6295984/65288900-7216d680-db40-11e9-9211-1f89939fc1e1.png">